### DISCUSSION TITLE
Typo

### DISCUSSION THREAD

**Discussion by rezgarshakeri** [mfem/mfem#2799] [General]
Hi @v-dobrev 
Shouldn't this equation be `4*pow(x(2), 3)`?

https://github.com/mfem/mfem/blob/a96065ed1802ee95e7a33ee749b2ae4a6e4a6de6/tests/unit/linalg/test_matrix_rectangular.cpp#L30

**Comment by tzanio** [comment]
Certainly looks like it should be 😄

**Reply by tzanio** [reply]
@rezgarshakeri, do you want to propose a simple PR to fix this?

**Reply by rezgarshakeri** [reply]
I think I already did, https://github.com/mfem/mfem/pull/2800#pullrequestreview-869539168

**Reply by tzanio** [reply]
My bad, sorry

**Reply by rezgarshakeri** [reply]
No problem, thanks.
